test/getiBFT_spec.rb

```diff
@@ -11,6 +11,7 @@
   describe "#getiBFT" do
     context "when filtering output of 'iscsiadm -m fw'" do
       it "returns data in form of a map " do
+        Yast::Arch.stub(:architecture).and_return("x86_64")
```

Comment:
Use the new rspec syntax here:

```ruby
allow(Yast::Arch).to receive(:architecture).and_return("x86_64")
```

The old syntax will stop working at some point in the future...